static/js/containers/ProfileFormContainer.js

```diff
@@ -98,7 +98,13 @@ class ProfileFormContainer extends React.Component {
         dispatch(clearProfileEdit(username));
       });
     } else {
-      return Promise.reject(errors);
+      return Promise.reject(errors).catch(() => {
```

Comment:
Hmm... one hacky way to solve this is to use `setState`: `setState({}, () => { // dom manipulation here })`. According to [the docs](https://facebook.github.io/react/docs/component-api.html) the `setState` callback triggers after component rerender. Right now there's no guarantee the `catch` callback will execute after rerender, but if the code is moved inside the `setState` callback we should have that guarantee.

Ideally we should do all DOM manipulation inside `componentDidMount` or `componentDidUpdate` or other relevant lifecycle methods, but doing it this way means we don't need to keep track of the state of whether we need to call `scrollIntoView`